swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -304,7 +262,7 @@ Follow the instructions given in this section to develop the service.
             log:printDebug("Email sent successfully",
                             name = patient.name,
                             appointmentNumber = appointmentNumber);
-            <http:Created>{};
+            return <http:Created>{};
         }
     }
 
```

Comment:
L293 onward, we decided to remove repeating the code snippets when explaining them.

Reply:
If we remove the code snippets, wouldn't it be harder to understand the descriptions?

Reply:
Since we describe in the order of the flow, should be OK without IMO.